src/dswx_sar/mosaic_gcov_frame.py

```diff
@@ -751,7 +751,7 @@ def run(cfg):
     # Mosaic input RTC into output Geotiff
     reader.process_rtc_hdf5(
         input_list,
-        output_dir,
+        scratch_dir,
```

Comment:
Just want to make sure that you wish to store everything in scratch directory.

Reply:
Yes. The intermediate files including mosaic_HH.tif should be in scratch directory.